### DISCUSSION TITLE
Globs in strict optional?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/pyright#8091] [Q&A] [answered]
I noticed that for the strict config option, a path to a directory  includes all its contents. For example: `strict = ["utils"]` will include all python files under `utils`. Is this behaviour intentional and going to stay?

<details><summary>Reference</summary>
<p>

> strict [array of paths, optional]: Paths of directories or files that should use “strict” analysis if they are included. This is the same as manually adding a “# pyright: strict” comment. In strict mode, most type-checking rules are enabled. Refer to [this table](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#diagnostic-settings-defaults) for details about which rules are enabled in strict mode. Paths may contain wildcard characters ** (a directory or multiple levels of directories), * (a sequence of zero or more characters), or ? (a single character).

</p>
</details>

**Comment by erictraut** [comment, marked as answer]
Yes, this behavior is intentional, and it has worked this way since the early days of pyright.
